test/MUIDataTableBody.test.js

```diff
@@ -263,4 +263,31 @@ describe('<TableBody />', function() {
     assert.isAtLeast(options.setRowProps.callCount, 1);
     assert(options.setRowProps.calledWith(data[1]));
   });
+
+  it("should use 'customRowRender' when provided", () => {
+    const options = { customRowRender: () => <div>Test_Text</div> };
+    const selectRowUpdate = stub();
+    const toggleExpandRow = () => {};
+
+    const t = mount(
```

Comment:
I believe elsewhere, we mostly use the name `fullWrapper` for a `mount` wrapped component, so it would be good to keep with that convention for clarity.

Reply:
I looked around in the repo and found that you use `fullWrapper` when you are mounting the "full" `<MuiDataTable/>`, but `mountWrapper` when you are wrapping only some sub-component. So I renamed `t` to `mountWrapper`, also in some other tests I found using `t` as a variable name: 26fdf818b249dc66623d8439548dd9483d4de1b1.